doc/changelog/04-tactics/16026-intuition-with-core.rst

```diff
@@ -0,0 +1,10 @@
+- **Deprecated:** the default ``intuition_solver`` (see
+  :tacn:`intuition`) now outputs warning ``intuition-auto-with-star``
+  if it solves a goal with ``auto with *`` that was not solved with
+  just :tacn:`auto`. In a future version it will be changed to just
```

Comment:
```suggestion
- **Deprecated:** the default ``intuition_solver`` (see
  :tacn:`intuition`) now generates the warning ``intuition-auto-with-star``
  if it solves a goal with ``auto with *`` that was not solved with
  just :tacn:`auto`. In a future version the default will be changed to
```

It would be a nice touch to hyperlink to where you added the message in tauto.ml.  The "Use the" can move to tauto.ml.

Reply:
Linking to the ocaml source in the user changelog sounds pretty weird.

Reply:
I meant to say tauto.rst.

Reply:
I duplicated this doc to the doc for intuition